bears/ruby/RuboCopBear.py

```diff
@@ -33,9 +33,11 @@ class RuboCopBear:
                     'convention': RESULT_SEVERITY.INFO}
 
     @staticmethod
```

Comment:
Remove the additional whitespaces. It's best if you run coala once before pushing the commit.